armi/reactor/converters/blockConverters.py

```diff
@@ -211,7 +211,12 @@ def convert(self):
 
 
 class ComponentMerger(BlockConverter):
-    """For a provided block, merged the solute component into the solvent component."""
+    """For a provided block, merged the solute component into the solvent component.
+
+    .. impl:: Homogenize one component into another
+        :id: I_ARMI_BLOCKCONV
+        :implements: R_ARMI_BLOCKCONV
```

Comment:
Great.

Can you ALSO add a crumb for this req to `MultipleComponentMerger`?  It's in this same file.